### PR TITLE
ast-json

### PR DESCRIPTION
**Pull request by freeton20**

### CODE REVIEW

**Review by elasticLove1 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -331,6 +331,34 @@ You can specify the output files location with the `-o` option:
...
+```shell
+tondev sol ast-json Contract.sol -o path/to/output/file
+```
+
+#### Ast-compact-json
+
+The same as previous but save AST in compact JSON form.
+
+```shell
+tondev sol ast-compact-json Contract.sol
```
Comment (melsomino) [inline README.md:355]: I think it is better to name command as `ast` and add option `-f, --format <json | compact-json>`

**Review by melsomino — CHANGES_REQUESTED** [review]
see comments

**Comment by elasticLove1** [comment]
@freeton20 How is it going with the requested changes?

**Comment by freeton20** [comment]
> @freeton20 How is it going with the requested changes?
Sorry for delay and new PR (playing with git:-)

https://github.com/tonlabs/tondev/pull/68

**Pull request closed without merging**